cypress/e2e/signIn.cy.js

```diff
@@ -2,6 +2,11 @@
 
 describe('Sign In page', () => {
   it('should provide an ability to log in', () => {
-
+    cy.visit('https://conduit.mate.academy/user/login');
+    cy.url().should('contain', 'https://conduit.mate.academy/user/login');
+    cy.get(':nth-child(1) > .form-control').type('bubasbwdgb@wdbg.chew');
```

Comment:
Avoid using selectors that are likely to change, like ':nth-child(1) > .form-control'. It's better to add data-* attributes to your HTML elements and use those for selecting. This way, your tests will not break with minor changes in your app.